src/vswhere.lib/LegacyInstance.cpp

```diff
@@ -0,0 +1,130 @@
+// <copyright file="LegacyInstance.cpp" company="Microsoft Corporation">
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt in the project root for license information.
+// </copyright>
+
+#include "stdafx.h"
+
+using namespace std;
+
+template <class T>
+inline HRESULT NotImplemented(_In_ T* p)
```

Comment:
That was my initial thought as well, but the method is to indicate its not really implemented while the convention is to return `E_NOTFOUND` if the property wasn't defined (at least for the original interfaces).